gramps/gui/plug/_windows.py

```diff
@@ -1217,7 +1217,7 @@ def install_addons(self, obj):
                      "%s %s" % (ngettext("{number_of} addon was installed.",
                                          "{number_of} addons were installed.",
                                          count).format(number_of=count),
-                                _("You need to restart Gramps to see new views.")),
+                        _("You need to restart Gramps to use the new addons.")),
```

Comment:
> @sam-m888 There is no global restart flag.

Thanks

For now I've only pushed the wording change mentioned above as https://github.com/gramps-project/gramps/commit/af674df82e33fc5e1ae76dc50e9cacf99e1666c4